config/config/sc-config.yaml

```diff
@@ -615,6 +615,9 @@ thanos:
     # standalone: Single instance of receiver.
     mode: dual-mode
 
+    # Can only be used if thanos.compactor.verticalCompaction is set to true
+    outOfOrderTimeWindow:  # 210s
+
```

Comment:
Also could you group this and the verticalcompaction variable together or at least put them next to each other, since this variable depends on that one?

Reply:
they seem to be under different keys so grouping them might not be possible, but I would like to try 10m for `outOfOrderTimeWindow`

Reply:
Sure i can increase it to 10 min i just had it at them minimum amount where i got no errors when testing.